packages/react-dom/rollup.config.js

```diff
@@ -0,0 +1,26 @@
+import fs from "fs";
+import plugin from "@eik/rollup-plugin";
+import { createRequire } from "module";
+
+const { resolve } = createRequire(import.meta.url);
+const reactPkg = new URL("../react/package.json", import.meta.url);
+const reactDomPkg = new URL("./package.json", import.meta.url);
+
+const { name, main } = JSON.parse(fs.readFileSync(reactPkg.pathname, "utf8"));
+const { version } = JSON.parse(fs.readFileSync(reactDomPkg.pathname, "utf8"));
+
+const importMap = {
+  imports: {
+    react: `https://assets.finn.no/npm/${name}/${version}/${main}`,
+  },
+};
+
+export default {
+  input: resolve("@esm-bundle/react-dom/esm/react-dom.production.min.js"),
```

Comment:
We need to use Node's resolve algorithm for this since I can't be 100% sure if the node_modules folder will be in this package, or in the root of the monorepo.